docs/report.md

```diff
@@ -35,7 +35,16 @@ Here comes a description of our domain model.
 ## How to make _Chirp!_ work locally
 
 ## How to run test suite locally
-
+The test suite of Chirp is divided in two test-projects. 'Chirp.Infrastructure.Tests' and 'Chirp.Web.Test'.
+In order to run all test suites of the Chirp application, you simply need to be located in the root directory (Chirp) and run the following command:
+```
+dotnet test
+```
+First this should run the UI Tests located in the 'Chirp.Web.Test' project, and should return a prompt with 3 tests passed.
+After that is will run the integrationtests from the 'Chirp.Infrastructure.Tests' project, which should return a promt with 26 tests passed.
```

Comment:
```suggestion
After that it will run the integration tests from the 'Chirp.Infrastructure.Tests' project, which should return a promt with 26 tests passed.
```